src/providers/websocket.ts

```diff
@@ -106,3 +106,6 @@ export class WebSocketProvider implements ApiProvider {
     });
   }
 }
+
+
+export { nunjucks, createTransformResponse };
```

Comment:
@gru-agent don't export nunjucks, you can import it from import { getNunjucksEngine } from '../util/templates'; instead

export createTransformResponse as `export function createTransformResponse(parser: any): (data: any) => ProviderResponse {` ...